### ISSUE TITLE
Resolve conflicts hangs (Windows 10)

### ISSUE THREAD

**Issue by photonstorm** [desktop/desktop#13363]
### Describe the bug

When the **Resolve Conflicts before Rebase** window opens and the 'Continue rebase' button is pressed, the app will hang and need to be closed via the Task Manager. When you then re-open it, the rebase will continue successfully and work as expected.

### Version & OS

Version 2.9.4 (x64)
Windows 10.0.19042

### Steps to reproduce the behavior

1. Perform a push/pull request that triggers a rebase
2. Resolve all of the conflicts - green check-marks appear next to each one
3. Click 'Continue rebase'

### Expected behavior

You shouldn't need to force-quit.

### Actual behavior

After clicking 'Continue rebase' GitHub Desktop now hangs. A forceful termination is required.

When re-opening GitHub Desktop the rebase completes successfully.

I'm not sure if it matters, but it seems to always happen when I pick 'Using the modified file from master' for all conflicted files.

This error happens consistently, it's not just a one-off.

### Screenshots

![image](https://user-images.githubusercontent.com/164476/142904052-2a808108-52b5-476d-a483-e36e62e41061.png)

### Logs

```
Rebasing (1/4)
error: could not apply 4f50af96e... Upgrade webpack
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 4f50af96e... Upgrade webpack

(The error was parsed as 6: We found some conflicts while trying to rebase. Please resolve the conflicts before continuing.)

// Crash was here

2021-11-22T16:59:13.026Z - info: [ui] Executing getStatus: git --no-optional-locks status --untracked-files=all --branch --porcelain=2 -z (took 2.066s)
2021-11-22T16:59:19.139Z - info: [ui] [AppStore] loading 106 repositories from store
2021-11-22T16:59:19.140Z - info: [ui] [AppStore] found account: photonstorm (Richard Davey)
2021-11-22T16:59:19.384Z - info: [ui] [BranchPruner] Last prune took place in a day - skipping
2021-11-22T16:59:19.519Z - info: [ui] launching: 2.9.4 (Windows 10.0.19042)
2021-11-22T16:59:19.519Z - info: [ui] execPath: 'C:\Users\rdave\AppData\Local\GitHubDesktop\app-2.9.4\GitHubDesktop.exe'
2021-11-22T16:59:29.631Z - info: [ui] [continueRebase] continuing rebase for ec4e8ccff7b909eb5cb93c9cf300d5086cbb65b8
2021-11-22T16:59:31.053Z - info: [ui] [continueRebase] completed rebase - got CompletedWithoutError and on tip 6c6084a4c52ca8d406ba15e4df7e90326020e4fc - kind Valid
2021-11-22T17:00:40.767Z - info: [ui] [Timing] Action 'create commit' for 'photonstorm/phaser' took 0.730s
2021-11-22T17:00:42.092Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: phaser - photonstorm (has token)
2021-11-22T17:00:44.438Z - info: [ui] Executing push: git -c credential.helper= -c protocol.version=2 push origin master:master --progress (took 2.266s)
```

**Comment by steveward** [comment]
Thanks for the detailed report @photonstorm. Are you able to consistently reproduce this crash? I'll take a look and see if I can reproduce it on my Windows machine as well.

**Comment by photonstorm** [comment]
> Thanks for the detailed report @photonstorm. Are you able to consistently reproduce this crash? I'll take a look and see if I can reproduce it on my Windows machine as well.

Yes, I can consistently reproduce it. I wouldn't waste your time if it was a one-off :)

It does take a rebase conflict to occur, which doesn't happen that often, but I could easily re-create one if you need another log?

**Comment by photonstorm** [comment]
Also worth mentioning - checking the task in Task Manager and it's completely idle, using 0% CPU and no network activity. So it doesn't _feel_ like it's stuck in some endless loop (although I guess it could be a very low CPU loop), more like it was expecting an event it never got.

Forcing the app to close and re-opening hasn't yet damaged anything or lost any data either, which is another reason I think it's just sat there idle, rather than actually processing.

**Comment by photonstorm** [comment]
I'm _pretty sure_ this issue is the same as #13204 which was closed in version 2.9.6. I'll keep an eye and see if it happens again!

**Comment by steveward** [comment]
Thanks @photonstorm -- it does look like that issue. I'm going to close this out since this should be resolved now.
